packages/driver/src/cypress/error_messages.coffee

```diff
@@ -773,7 +773,9 @@ module.exports = {
     invalid_capture: "{{cmd}}() 'capture' option must be one of the following: 'fullPage', 'viewport', or 'runner'. You passed: {{arg}}"
     invalid_boolean: "{{cmd}}() '{{option}}' option must be a boolean. You passed: {{arg}}"
     invalid_blackout: "{{cmd}}() 'blackout' option must be an array of strings. You passed: {{arg}}"
-    invalid_clip: "{{cmd}}() 'clip' option must be an object of with the keys { width, height, x, y } and number values. You passed: {{arg}}"
+    invalid_clip: "{{cmd}}() 'clip' option must be an object with the keys { width, height, x, y } and number values. You passed: {{arg}}"
+    invalid_height: "#{cmd('screenshot')} only works with a screenshot area whose height is greater than zero."
```

Comment:
@jennifer-shehane I've reworded this to talk about the `height` only because it's the only thing we can make any judgments about when checking `numScreenshots` or `scrolls.length` (which is derived from `numScreenshots`)

Reply:
I think this is okay, but "whose height is greater than zero" is weird to me (because I normally use the word 'whose' to refer to people). I would have written "with a height greater than zero".